examples/MilvusClientExample.go

```diff
@@ -0,0 +1,215 @@
+package main
+
+import (
+	"github.com/milvus-io/milvus-sdk-go/milvus"
+	"strconv"
+	"time"
+)
+
+var tableName string = "test_go"
+var dimension int64 = 128
+var indexFileSize int64 = 1024
+var metricType int64 = int64(milvus.L2)
+var nq int64 = 100
+var nprobe int64 = 64
+var nb int64 = 100000
+var topk int64 = 100
+var nlist int64 = 16384
+
+func example(address string, port string) {
+	var grpcClient milvus.Milvusclient
```

Comment:
4 characters tab is OK, why they are 8 characters?

Reply:
go fmt set=8 defaul. This show 8 characters in github repo.